README.md

```diff
@@ -197,7 +198,7 @@ To use the GraphDB module, create a new Terraform project or add to an existing
 ```hcl
 module "graphdb" {
   source  = "Ontotext-AD/graphdb/azure"
-  version = "1.0.0"
+  version = "1.2.0"
```

Comment:
We can avoid updating this if we use something like
```
version = "~> 1"
```
see https://developer.hashicorp.com/terraform/tutorials/configuration-language/versions#terraform-version-constraints